src/plugins/auctionHouseModule/createBid.ts

```diff
@@ -46,9 +94,110 @@ export type CreateBidBuilderParams = Omit<CreateBidInput, 'confirmOptions'> & {
   instructionKey?: string;
 };
 
+export type CreateBidBuilderContext = Omit<CreateBidOutput, 'response'>;
+
 export const createBidBuilder = (
   metaplex: Metaplex,
   params: CreateBidBuilderParams
-): TransactionBuilder => {
-  return TransactionBuilder.make();
+): TransactionBuilder<CreateBidBuilderContext> => {
+  // Data.
+  const auctionHouse = params.auctionHouse;
+  const tokens = params.tokens ?? token(1);
+  const priceBasisPoint = params.price?.basisPoints ?? 0;
+  const price = auctionHouse.isNative
+    ? lamports(priceBasisPoint)
+    : amount(priceBasisPoint, auctionHouse.treasuryMint.currency);
+
+  // Accounts.
+  const wallet = params.wallet ?? (metaplex.identity() as Signer);
+  const authority = params.authority ?? auctionHouse.authorityAddress;
+  const metadata = findMetadataPda(params.mintAccount);
+  const tokenAccount =
+    params.tokenAccount ??
+    findAssociatedTokenAccountPda(params.mintAccount, toPublicKey(wallet));
```

Comment:
This is wrong. The token account refers to the token account of the seller so it makes no sense to create an ATA with the address of the buyer.

As mentioned above, this could also be `null` for public bids.